packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -72,8 +72,12 @@ $link-hover-color: darken($link-color, 15%);
 //== Typography
 //## Change your font family, weight, line-height, headings and more (used in components/typography)
 
-// Font Family Import (Used for google font plugin in theme creater)
+// Font Family Import (Used for google font plugin in theme creater https://ux.mendix.com/theme-creator.html)
 $font-family-import: "https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700";
+@if $font-family-import != false {
```

Comment:
This no longer happens here, see this PR: https://github.com/mendix/widgets-resources/pull/493

How ome you've added it in?

Reply:
Removed the import in this file. It already happens in `main.scss` now.